docs/chaos-engineering/technical-reference/chaos-faults/aws/static/manifests/ecs-task-scale/task-service-name.yaml

```
@@ -0,0 +1,23 @@
+# scale the tasks of an ECS cluster
+apiVersion: litmuschaos.io/v1alpha1
+kind: ChaosEngine
+metadata:
+  name: engine-nginx
+spec:
+  engineState: "active"
+  annotationCheck: "false"
+  chaosServiceAccount: litmus-admin
+  experiments:
+  - name: ecs-task-scale
+    spec:
+      components:
+        env:
+        # provide the name of ECS cluster
+        - name: CLUSTER_NAME
+          value: 'demo'
+        - name: SERVICE_NAME
+          vale: 'test-svc'
+        - name: REGION
```

```suggestion
          value: 'demo'
        - name: SERVICE_NAMES
          vale: 'test-svc'
        - name: REGION
```

Please synk with the YAML in the main docs page.